src/core/shape_inference/include/fft_common_validation.hpp

```diff
@@ -91,9 +91,9 @@ void validate_axes(const ov::op::util::FFTBase* op,
     for (int64_t& axis : axes) {
```

Comment:
As part 2 (improvments): Check if this loop can be replaced by `normalize_axes` or use `normalize_axis` from validation_util.